db/seeds.rb

```diff
@@ -190,13 +190,16 @@ def create_case(full_name,
     end
     
     start_time = Tod::TimeOfDay.new(rand(0..10), ((rand(0..59) / 15.0).floor * 15))
-
-    Schedule.find_or_initialize_by(child: child).update!(
-      effective_on: Faker::Date.between(from: 8.months.ago, to: 4.months.ago),
-      end_time: start_time + rand(start_time.hour..23).hours,
-      start_time: start_time,
-      weekday: rand(0..6)
-    )
+    effective_on = Faker::Date.between(from: 8.months.ago, to: 4.months.ago)
+
+    5.times do |idx|
```

Comment:
Make a more realistic schedule for the kids being onboarded via seeding, rather than with a default schedule